pkg/services/sqlstore/migrator/migrator.go

```diff
@@ -400,6 +400,11 @@ func (mg *Migrator) InTransaction(callback dbTransactionFunc) error {
 	sess := mg.DBEngine.NewSession()
 	defer sess.Close()
 
+	// XXX: Spanner cannot execute DDL statements in transactions
+	if mg.Dialect.DriverName() == Spanner {
+		return callback(sess)
+	}
```

Comment:
I expect this to be removed once the migrator is able to run a squashed snapshot of migrations